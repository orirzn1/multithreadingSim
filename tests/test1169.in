L8
S5
O13
N6

T0
I@0x00000000
STORE $4, $4, $7
STORE $3, $7, 0x0
SUB $6, $2, $0
ADDI $0, $3, 0x4
ADD $4, $0, $7
SUBI $5, $1, 0x1
STORE $1, $7, $1
LOAD $5, $4, 0x3
STORE $5, $7, 0x1
SUB $6, $0, $0
HALT $0

T1
I@0x00000000
SUBI $1, $0, 0x2
ADD $4, $2, $2
ADDI $7, $4, 0x2
SUB $4, $1, $7
LOAD $5, $7, $2
LOAD $3, $0, 0x0
SUBI $7, $4, 0x3
SUBI $0, $1, 0x3
HALT $0

T2
I@0x00000000
SUBI $2, $1, 0x3
LOAD $1, $1, 0x2
SUBI $6, $5, 0x3
SUB $0, $5, $0
STORE $3, $3, 0x3
STORE $5, $0, $5
ADDI $0, $3, 0x2
HALT $0

T3
I@0x00000000
STORE $0, $5, $4
LOAD $6, $5, 0x4
LOAD $3, $4, $7
SUBI $5, $7, 0x1
HALT $0

T4
I@0x00000000
STORE $1, $0, 0x0
ADDI $2, $4, 0x1
LOAD $5, $5, $1
ADD $0, $1, $5
HALT $0

T5
I@0x00000000
LOAD $1, $5, 0x3
ADDI $2, $1, 0x4
LOAD $1, $6, $5
ADD $4, $7, $1
HALT $0

D@0x0
0x8
0x5
0x4
0x4
0x3
0x5
