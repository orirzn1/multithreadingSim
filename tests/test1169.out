
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x4	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x5	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFB	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFC	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFF9
Register file thread id 2:
	R0 = 0x2	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x5	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 7.465116

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x4	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFB	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFC	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFF9
Register file thread id 2:
	R0 = 0x2	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.186047

